cmd/build/helmify/static/Chart.yaml

```diff
@@ -3,7 +3,7 @@ description: A Helm chart for Gatekeeper
 name: gatekeeper
 keywords:
   - open policy agent
-version: v3.1.0-rc.1
+version: 3.1.0-rc.1
```

Comment:
why do we want to change this? the current chart.yamls all have this

Reply:
Helm requires semver2 which doesn't include `v`. https://helm.sh/docs/topics/charts/#charts-and-versioning